components/cdc/src/observer.rs

```diff
@@ -191,7 +191,7 @@ impl RoleObserver for CdcObserver {
                 let deregister = Deregister::Region {
                     region_id,
                     observe_id,
-                    err: CdcError::Request(store_err.into()),
+                    err: CdcError::request(store_err.into()),
```

Comment:
Can the changes to error be extracted as a separate PR?

Reply:
We need these changes because clippy complain about `large_enum_variant` after we add `DataIsNotReady` to `kvproto::errorpb::Error` which is a variant of `CdcError`